custom/icds_reports/models/aggregate.py

```diff
@@ -1442,6 +1470,11 @@ class AggregateCcsRecordDeliveryForms(models.Model):
         help_text="number of qualified visits for the incentive report",
         default=0
     )
+    where_born = models.PositiveSmallIntegerField(
+        null=True,
+        help_text="Where the child is born",
+        default=0
```

Comment:
I mentioned this on a separate PR, but defaults are dangerous because they lock the table. can you change this, and re-run the migration to make sure a database default is not created. (I think you need to merge master and re-run the migration anyways due to migration conflicts

Reply:
Removed the default 0 @emord